### PR TITLE
Feature/2817 lightning 2017 q1

### PR DESCRIPTION
**Pull request by chrisarusso**

### CODE REVIEW

**Comment by lrowang** [comment]
This PR is extension of #225 .

File: _posts/2017-03-20-lightning-q1-2017.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+featured_image_height: "917px"
+featured_image_width: "1474px"
+summary: What we've added to our developer toolboxes in Q1 2017!
+---
+
+_This post is part of a new series called "Lightning Talks."_
+
+The Web Development world evolves quickly, and keeping abreast of new tools and methodologies while simultaneously delivering client work is tough.
+
+One of our core [philosophies](http://savaslabs.com/mission-and-values/#excel) at Savas Labs is to "_hold ourselves to a high and constantly improving standard_." To that end, we dedicate time each month to share new tips and tools with each other to continue increasing our collective knowledge base -- we call these our :zap: "Lightning Skillshares!" :zap:
```
Comment (kostajh) [inline _posts/2017-03-20-lightning-q1-2017.md]: @lrowang should use a relative link, `/mission-and-values/#excel`

File: _posts/2017-03-20-lightning-q1-2017.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+featured_image: "/blog/lightning-17q1-hero.jpg"
+featured_image_alt: "Q1 2017 Lightning Talks"
+featured_image_height: "917px"
+featured_image_width: "1474px"
+summary: What we've added to our developer toolboxes in Q1 2017!
+---
+
+_This post is part of a new series called "Lightning Talks."_
+
+The Web Development world evolves quickly, and keeping abreast of new tools and methodologies while simultaneously delivering client work is tough.
```
Comment (kostajh) [inline _posts/2017-03-20-lightning-q1-2017.md]: consider lowercasing "Web Development"? not sure if it should be capitalized, I usually don't.

File: _posts/2017-03-20-lightning-q1-2017.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+It works by starting an SSH server on the host computer and generating a unique session ID that others can connect to via their terminal or a web browser. You can have multiple connected individuals in a session, though we typically use it more for 1-1 debugging and troubleshooting.
+
+Be aware, collaborators will have _direct_ access to your terminal, so only share with people you trust and be careful about accidental commands. Other than that, it's free and easy to set up -- check it out [here](https://www.teleconsole.com/).
+
+<script type="text/javascript" src="https://asciinema.org/a/7oargr7f911dm8qz4c5m0k2iw.js" id="asciicast-7oargr7f911dm8qz4c5m0k2iw" async></script>
+<span class="caption">Terminal sharing 2,700 miles away!</span>
+
+## 2. Accessing a Dockerized database via PhpMyAdmin or PhpStorm
+
+There are many useful tools available for interacting with a MySQL database. Since we've transitioned to Docker for our local development, we prefer one of the following techniques for manipulating databases abstracted into the Docker container.
```
Comment (kostajh) [inline _posts/2017-03-20-lightning-q1-2017.md]: maybe link "Docker for our local development" to our Docker tag on the website

File: _posts/2017-03-20-lightning-q1-2017.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+1. Simply add this snippet from docker4drupal to your `docker-compose.yml` file
+  - It uses the official `phpmyadmin` image and defines the host as the database container (`mariadb` in this example), so be sure to add it _after_ the section defining your database container.
+<img src="/assets/img/blog/lightning-17q1-phpmyadmin.png" class="blog-image-large" alt="phpmyadmin container">
+<span class="caption">Add to `docker-compose.yml`</span>
+2. Set the correct database user and password (`PMA_USER` and `PMA_PASSWORD`).
+
+Once the containers are up you can visit the port specific url (for instance `<example.site>:8001`) to access your project-specific PhpMyAdmin interface! :tada:
+
+### Using PhpStorm to interact with a Dockerized Database
+Another method is to connect PhpStorm directly to the database container using PhpStorm's Database Browser. Instead of SSHing into the container via the command line, this allows us to view DB tables and run SQL queries directly within the PhpStorm interface!
```
Comment (kostajh) [inline _posts/2017-03-20-lightning-q1-2017.md]: as noted earlier, replace "SSHing" with "opening a shell"

File: _posts/2017-03-20-lightning-q1-2017.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+1. In PhpStorm, go to the "Database Browser", click the `+` dropdown and select "Data Source > MySQL"
+<img src="/assets/img/blog/lightning-17q1-pstorm1.png" class="blog-image-large" alt="phpstorm connect1">
+2. In the popup window:
+  - Map the project DB container's port specified in the `docker-compose.yml` file to `Port` (in the below example it's `32770`)
+  - Add the `Database`, `User`, and `Password` fields from the `docker-compose.yml` file as well
+  - You can select `Test Conneciton` or just `OK`
+<img src="/assets/img/blog/lightning-17q1-pstorm2.png" class="blog-image-xl" alt="phpstorm connect2">
+
+Once connected, PhpStorm will display all the tables in a drop down in the Database Browser, which is useful for quickly glancing through all the tables! You'll also be able to run SQL queries within PhpStorm and see outputted results without having to leave the interface! :star2:
```
Comment (kostajh) [inline _posts/2017-03-20-lightning-q1-2017.md]: and you get useful code completions when writing database queries

File: _posts/2017-03-20-lightning-q1-2017.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+
+## 3. Using Apowersoft's Free Screen Recorder to capture your screen
+
+If a picture's worth a thousand words, than a video is worth...a million? There are times when it's useful to send a short video (for example, a quick tutorial to a client) and Apowersoft's [Free Online Screen Recorder](https://www.apowersoft.com/free-online-screen-recorder) is a great tool for capturing your screen. It's easy to use and there's _no recording time limit_!  Other things we like are the ability to customize the cursor appearance, limit the capture to a specific region of your screen, and specify the resulting video format.
+
+Setup is incredibly easy:
+- Go to the [site](https://www.apowersoft.com/free-online-screen-recorder) and download the Launcher (you only have to do this once)
+- Now anytime you want to record, just go back to the page and hit "Start Recording"!
+
+*There is a Desktop version for those who prefer using the full application. However the free desktop version limits recordings to 3 minutes.
```
Comment (kostajh) [inline _posts/2017-03-20-lightning-q1-2017.md]: need to close the italics on this sentence

**Review by kostajh — CHANGES_REQUESTED** [review]
Only "required" change from me is to use the relative link for a post on our site. Other than that, nice work!

File: _posts/2017-03-20-lightning-q1-2017.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+<script type="text/javascript" src="https://asciinema.org/a/7oargr7f911dm8qz4c5m0k2iw.js" id="asciicast-7oargr7f911dm8qz4c5m0k2iw" async></script>
+<span class="caption">Terminal sharing 2,700 miles away!</span>
+
+## 2. Accessing a Dockerized database via PhpMyAdmin or PhpStorm
+
+There are many useful tools available for interacting with a MySQL database. Since we've transitioned to [Docker for our local development](/blog/tag/docker/), we prefer one of the following techniques for manipulating databases abstracted into the Docker container.
+
+### Adding PhpMyAdmin to a Docker Stack
+
+[PhpMyAdmin](https://www.phpmyadmin.net/) is a great tool that provides a useful interface for visualizing and interacting with a MySQL database. [Docker4drupal](https://github.com/wodby/docker4drupal) has created an easy [template](https://github.com/wodby/docker4drupal/blob/master/docker-compose.yml) for adding the PhpMyAdmin service to a project's Docker Stack.
```
Comment (dmurphy1) [inline _posts/2017-03-20-lightning-q1-2017.md]: Hi @lrowang - Docker4Drupal has unfortunately changed their template so can we remove the referenes to Docker4Drupal in this post? The PHPMyAdmin container isn't dependent on Docker4Drupal at all, I just got my inspiration from them! If you're curious, this is the [latest Docker4Drupal template](https://github.com/wodby/docker4drupal/blob/master/docker-compose.yml#L61).

Comment (lrowang) [reply _posts/2017-03-20-lightning-q1-2017.md]: Thanks for catching that @dmurphy1 ! I made the fix!

**Comment by lrowang** [comment]
Deploying for now! Using basic asciinema video for now and we'll revisit the updated teleconsole gif/video soon!

**Pull request merged**